### DISCUSSION TITLE
how back to search result

### DISCUSSION THREAD

**Discussion by karb10** [Flow-Launcher/Flow.Launcher#2861] [Q&A]
![image](https://github.com/user-attachments/assets/66ca2118-4846-4695-a268-950aa40f941e)
how  back to ![image](https://github.com/user-attachments/assets/6f3f3717-18b5-40bb-ac72-c8bc1e1c2ff7) again?

**Comment by deefrawley** [comment]
Hit ESCAPE

**Reply by karb10** [reply]
ESCAPE closing windows complete, i need back to search result (second screen)

**Reply by deefrawley** [reply]
I thought you were referring to the context menu

**Comment by Garulf** [comment]
Sorry there is no "back" button.

**Comment by jjw24** [comment]
Press Ctrl+Z for undo I suppose is one way.

**Reply by z1nc0r3** [reply]
CTRL + Z also does not work when we use the _tab_ to forward to a path. But other times we can use undo.
